content/docs/gke/private-clusters.md

```diff
@@ -24,6 +46,165 @@ Before installing Kubeflow ensure you have installed the following tools:
   * [gcloud](https://cloud.google.com/sdk/)
   * [jsonnet](https://github.com/google/jsonnet/releases)
 
+
+You will need to know your gcloud organization id and project number; you can get it via gcloud
```

Comment:
I think we need to explain here that a project number is different from a project ID, since this page uses both. Here's a useful doc to link to:
https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects

Reply:
Done.